api/src/test/java/org/openmrs/module/dhisreport/api/importsummary/importsummariesTest.java

```
@@ -0,0 +1,101 @@
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.bind.Unmarshaller;
+
+import org.junit.Test;
+import org.openmrs.module.dhisreport.api.importsummary.ImportConflict;
+import org.openmrs.module.dhisreport.api.importsummary.ImportCount;
+import org.openmrs.module.dhisreport.api.importsummary.ImportSummaries;
+import org.openmrs.module.dhisreport.api.importsummary.ImportSummary;
+
+public class importsummariesTest {
+
+	
+		
+	
+	@Test
+	public void test() throws JAXBException {
+		ImportConflict imconfl = new ImportConflict("object1", "value1");
+		ImportCount imco = new ImportCount();
+		int imported = 1;
+		int ignored = 2;
+		int updated = 3;
+		
+		imco.setImported(imported);
+		imco.setIgnored(ignored);
+		imco.setUpdated(updated);
+	
+		ImportSummary is = new ImportSummary();
+		is.setDataSetComplete("dataSetComplete");
+		is.setDescription("description");
+		is.setImportCount(imco);
+		is.setStatus("status");
+		List<ImportConflict> ic = new ArrayList<ImportConflict>();
+		ic.add(imconfl);
+		is.setConflicts(ic);
+		
+		ImportSummaries iss = new ImportSummaries();
+		int deleted =4;
+		
+		iss.setDeleted(deleted);
+		iss.setIgnored(ignored);
+		iss.setImported(imported);
+		iss.setUpdated(updated);
+		List<ImportSummary> isum = new ArrayList<ImportSummary>();
+		isum.add(is);
+		iss.setImportSummaryList(isum);
+		
+		Writer xmlWriter = new StringWriter();
+        JAXBContext jaxbContext = null;
+		jaxbContext = JAXBContext.newInstance( ImportSummaries.class );
+        Marshaller jaxbmarshaller = jaxbContext.createMarshaller();
+        jaxbmarshaller.marshal( iss, xmlWriter );
+        String xml = xmlWriter.toString();
+        
+        InputStream is1 = null;
+		try {
+			// Convert the String into InputStream
+			is1 = new ByteArrayInputStream(xml.getBytes("UTF-8"));
+
+		} catch (Exception e) {
+			e.printStackTrace();
```

Don't we just wanna let this bubble through instead of catching it here?